packages/interface/src/providers/index.tsx

```diff
@@ -30,22 +31,41 @@ const customTheme: Theme = {
   },
 };
 
+const defaultPrivyAppId = "cm38ydfhg01tggniibp652pmz";
+
 export const Providers = ({ children }: PropsWithChildren): JSX.Element => {
   const { config, queryClient } = useMemo(() => createWagmiConfig(), []);
 
   return (
     <ThemeProvider attribute="class" forcedTheme={appConfig.theme.colorMode}>
       <WagmiProvider config={config}>
         <QueryClientProvider client={queryClient}>
-          <RainbowKitProvider theme={customTheme}>
-            <RoundProvider>
-              <MaciProvider>
-                <BallotProvider>{children}</BallotProvider>
+          <PrivyProvider
+            appId={process.env.NEXT_PUBLIC_PRIVY_APP_ID ?? defaultPrivyAppId}
```

Comment:
oh I see it's here, so do we need this `defaultPrivyAppId`? What is it?

Reply:
doesn't build without it, the key is not valid but follows the correct format